src/chain/tracker.rs

```diff
@@ -324,30 +324,30 @@ impl ChainWatcher {
             &chain.asset
         );
         // Remove unwanted assets
-        assets.retain(|asset_name, properties| {
-            tracing::info!(
-                "chain {} has token {} with properties {:?}",
-                &chain.name,
-                asset_name,
-                properties
-            );
+        assets = assets
```

Comment:
can you elaborate why thats necessary?
per what I remember we discussed in one of the previous PRs and decided its better to do retain

Reply:
https://github.com/Alzymologist/Kalatori-backend/pull/110#discussion_r1835068551
https://github.com/Alzymologist/Kalatori-backend/pull/110#discussion_r1835086736

Reply:
The previous code don't enable custom names for assets & take asset names from a chain. With `retain`, it's impossible to mutate the key value, i.e. the asset name. The current code allow custom names & take asset names from the config. What else needs to be explained?

Reply:
Why do we need asset names that are different from chain? Won't this promote even more confusion when things have several different names?